CHANGELOG.md

```diff
@@ -10,19 +10,42 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 ### Added
+
+
+[v0.18.4][v0.18.4_log]
+--------------
+### Added
 - `oe_set_host_log_level(oe_log_level_t log_level)` allows applications to dynamically modify the verbosity of logs captured from the host.
   - It can be called with or without logging callback set and is thread safe. There is no limit on the number of calls per application.
 
 - `oe_set_enclave_log_level(oe_enclave_t* enclave, oe_log_level_t log_level)` is a host API that allows an application to dynamically modify the verbosity of logs generated by the given `enclave`.
   - This API is enabled only when logging.edl is imported.
   - This API can be used only after enclave creation and before enclave termination for an enclave. An application can call this API any number of times, for any number of enclaves. There is no limit on the number of calls and is thread safe.
   - This is an host API. To modify verbosity of enclave log level from within enclaves, consider using enclave logging callbacks.
-  - Updated the mbedTLS from 2.28.0 to 2.28.1
-## Changed
-- Fix the incorrect behavior of pthread_mutex_init() and std::mutex such that they no longer create a recursive lock by default.
+- Added backtrace debug log for unhandled exceptions if the enclave is configured with and CapturePFGPExceptions=1, the backtrace information will be printed in the enclave log when an in-enclave exception is not handled by trusted handlers. If the enclave is not configured with CapturePFGPExceptions=1, the similar helper message in the following will be printed in the log: `2022-07-13T00:25:05+0000.276579Z  (H)ERROR] tid(0x7f6cbb2b1f40) | Unhandled in-enclave exception. To get more information, configure the enclave with CapturePFGPExceptions=1 and enable the in-enclave logging.`
```

Comment:
Added backtrace debug log for unhandled exceptions if the enclave is configured with ~and~ CapturePFGPExceptions=1